proposals/low-level-struct-improvements.md

```diff
@@ -431,6 +372,22 @@ Detailed Notes:
     - The core library contains the feature flag indicating support for `ref` fields
     - The `langversion` value is 11 or higher
 
+### Sunset restricted types
+The compiler has a concept of a set of "restricted types" which is largely undocumented. These types were given a special status because in C# 1.0 there was no general purpose way to express their behavior. Most notably the fact that the types can contain references to the execution stack. Instead the compiler had special knowledge of them and restricted their usage in ways that would always be safe: disallowed returns, cannot use as array elements, cannot use in generics, etc ...
```

Comment:
>restricted their usage in ways that would always be safe [](http://example.com/codeflow?start=346&length=56)

Perhaps "restricted their **use to** ways that would always be safe" or "prevented their use in ways that would be unsafe".